CHANGELOG.md

```diff
@@ -6,6 +6,17 @@
 
 * It is now possible to supply custom `google-labels` in [workflow options](https://cromwell.readthedocs.io/en/stable/wf_options/Google/).
 
+### AWS backend
+
+It is now possible to use WDL disk attributes with the following formats on AWS.
+```
+disks: "local-disk 20 SSD"
+```
+```
+disks: "/some/mnt 20 SSD"
+```
+Because AWS auto-sizes disks, the size specification is simply discarded.
```

Comment:
The type (SSD vs. HDD) is also discarded? Also, I'd put this info in the docs, and just put a link-to-the-docs here in the changelog.

Reply:
It's definitely not used right now because on `develop` AWS only accepts `local-disk` or `/some/mnt` without any disk information.

I am trying to dig up AWS documentation on what a default might be, but no results yet.

Reply:
Actually, I'd also like to confirm the "auto-size" point - I can't seem to find a source for it online.

Perhaps @TMiguelT can chime in on what AWS Batch does with respect to disks, when no information can be provided through workflow attributes.

Reply:
@kshakir This is why WDL runtime is stupid. It makes no sense in a generic angle. And since the key WDLs ppl want to run have a very google specific string embedded ......

Reply:
Clarified, documented, and ready for re-review.

Reply:
> This is why WDL runtime is stupid.

PRs welcome /s